pages/linux/a2disconf.md

```diff
@@ -0,0 +1,12 @@
+# a2disconf
+
+> Disable an Apache configuration file on Debian-based OSes.
+> More information: <https://httpd.apache.org/>.
```

Comment:
Is there a more specific link to the documentation about this command? Same for the other pages.

Reply:
Since this command was created for Debian, we could use their man page: https://manpages.debian.org/buster/apache2/a2disconf.8.en.html

Reply:
Ah, that'd be perfect! That's do that.